catalog/solr/catalog-solr-offline-gazetteer/pom.xml

```diff
@@ -14,8 +14,8 @@
 
  -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
NOTE: This will be easier to review if you ignore whitespace.

Reply:
🤯